packages/office-ui-fabric-react/src/common/DocPage.types.ts

```diff
@@ -99,6 +99,9 @@ export interface IDocPageProps {
   /** Best practice as markdown string */
   bestPractices?: string;
 
+  /** Feedback section includes link to new issue page */
```

Comment:
As this is a Boolean, the name should ideally reflect it, so `renderFeedback: boolean` or `hasProvideFeedback: Boolean` for instance.

Reply:
@anavilohia this is solid advice. booleans that aren't html properties (disabled, checked etc) should indicate that they are a yes/no proposition. so hasFeedback would be prefered (just like isHeaderVisible) below.